sky/backends/backend_utils.py

```diff
@@ -2004,7 +2005,7 @@ def refresh_cluster_record(
     *,
     force_refresh_statuses: Optional[Set[status_lib.ClusterStatus]] = None,
     acquire_per_cluster_status_lock: bool = True,
-    acquire_lock_timeout: int = CLUSTER_FILE_MOUNTS_LOCK_TIMEOUT_SECONDS
+    acquire_lock_timeout: int = CLUSTER_STATUS_LOCK_TIMEOUT_SECONDS
```

Comment:
This naming made me wonder what lock it is, is it the same as L2007. Can we rename it to `acquire_per_cluster_status_lock_timeout`? Need to add to Args too.